.expeditor/verify.pipeline.yml

```diff
@@ -69,7 +69,9 @@ steps:
 
   - label: "[unit] applications-service"
     command:
-      - hab studio run "source .studiorc && go_component_unit applications-service && go_component_static_tests applications-service"
+      - scripts/install_golang
+      - cd components/applications-service
+      - make ci
```

Comment:
the make tasks should be runnable inside or out, as anyone prefers. Personally, I use the studio for anything involving actually running a service but will run unit and static stuff outside and for some changes to the deployment service you pretty much need to run them in vagrant (stuff that interacts with systemd, etc.). As for changing `go_component_static_tests` that would globally change things for the 3 other projects using it, whereas the benefit of (also one of the problems with) the makefile approach is it can be customized to what teams want.